src/middlewared/middlewared/plugins/auth.py

```diff
@@ -440,9 +444,16 @@ async def login(self, app, username, password, otp_token):
         """
         user = await self.get_login_user(username, password, otp_token)
         if user is None:
+            await app.log_audit_message("AUTHENTICATION", {
+                "credentials": {
+                    "credentials": "LOGIN_PASSWORD",
+                    "credentials_data": {"username": username},
+                },
+                "error": "Invalid username",
```

Comment:
This can fail due to either bad username or bad password. We should either distinguish between the two cases or state that it's either the username or password at fault.

Reply:
Maybe "Bad username or password".